service/src/test/java/bio/terra/externalcreds/TestUtils.java

```diff
@@ -68,4 +83,55 @@ public static VisaVerificationDetails createRandomVisaVerificationDetails() {
         .visaJwt(UUID.randomUUID().toString())
         .build();
   }
+
+  public static SshKeyPair createRandomGithubSshKey() throws NoSuchAlgorithmException, IOException {
+    var randomExternalUserEmail =
+        RandomStringUtils.random(5, /*letters=*/ true, /*numbers=*/ true) + "@gmail.com";
+    var keyPair = generateRSAKeyPair();
```

Comment:
FWIW, I am not a fan of using `var` where the type of the right hand side is not obvious.